src/mono/wasm/runtime/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "noImplicitAny": true,
+    "noEmitOnError": true,
+    "removeComments": false,
+    "sourceMap": false,
```

Comment:
Is there a reason why you turned off source maps?

Reply:
There are 4 steps which map has to survive in sync with the code positions.
1) typescript transpilation
2) rollup-plugin-terser
3) emcc (closure)
4) AOT and customer AOT

I thought I will leave it for later as it seems like rabbit hole.